engine/runtime/src/main/java/org/enso/interpreter/node/callable/resolver/HostMethodCallNode.java

```diff
@@ -48,6 +48,26 @@ public enum PolyglotCallType {
      * Standard.Base.Data.Time.Date} and dispatching natively.
      */
     CONVERT_TO_DATE,
+    /**
+     * The method call should be handled by converting {@code self} to a {@code
+     * Standard.Base.Data.DateTime} and dispatching natively.
+     */
+    CONVERT_TO_ZONEDDATETIME,
```

Comment:
Nitpick: it should probably be `CONVERT_TO_ZONED_DATE_TIME` when the class is called `ZonedDateTime`. Similar for other constants. Not that it would affect the functionality, however.